doc/tutorials/mediasdk-tutorials-cmd-reference.md

```diff
@@ -103,8 +116,22 @@ Tutorial demonstrates how to decode given raw video stream (input-file) of H.264
 | ------:| -----------:|
 | input-file | Mandatory argument. Sets incoming input bitstream to process. Input data should be in raw H.264 format. |
 | output-file | Optional argument. Sets uncompressed video file to write output data. If file will not be specified, tutorial will work in the performance mode: it will process data, but will not produce any output. |
+## simple_2_decode_hevc10
+**simple_decode *[-sw | -hw | -auto] input-file [ output-file ]***
+Tutorial demonstrates how to decode given raw video stream (input-file) of H.265 format. If output-file was specified, decoded YUV will be written into it. If the output-file is omitted, tutorial can be used to estimate constructed pipeline’s performance. Decoder is configured to produce decoded data in the system memory.
```

Comment:
If output-file was specified -> If output-file is specified

Reply:
done